src/firefly/js/ui/dynamic/EmbeddedPositionSearchPanel.jsx

```diff
@@ -154,7 +154,7 @@ export function EmbeddedPositionSearchPanel({
                     placeholder: 'Coordinates',
                     manageHiPS:false,
                 }} />}
-            {otherComponents}
+            {otherComponents && otherComponents(doGetConeAreaOp())}
```

Comment:
You don’t need to change this file at all. The components are part of the same field group so they can use.

```js
const [getConeAreaOp, setConeAreaOp] = useFieldGroupValue(CONE_AREA_KEY);
```

Reply:
@robyww if you get a chance before you leave (or on Wednesday) to take a look at this (the [IFE PR](https://github.com/IPAC-SW/irsa-ife/pull/326) is updated with latest code changes)...I've kept this code as is, as I was unable to use `useFieldGroupValue` in the `otherComponents` defined in `RegionExplorer.jsx`, as it's not a React Component. 
I may be missing an obvious trick here, but let me know if you figure out how to do this.